openshift/s2i/nginx-runtime/nginx.conf.template

```diff
@@ -40,7 +40,7 @@ http {
         ignore_invalid_headers off;
 
         # add in most common security headers
-        add_header Content-Security-Policy "default-src 'self'; frame-ancestors 'self'; img-src 'self' data: https://openmaps.gov.bc.ca/ https://*.tile.openstreetmap.org/ https://server.arcgisonline.com/; frame-src 'self' https://*.oidc.gov.bc.ca; connect-src 'self' https://*.oidc.gov.bc.ca/ https://openmaps.gov.bc.ca/ https://server.arcgisonline.com/; form-action 'self'; style-src 'self' 'sha256-TJWtAQZhIjMP4+e4lQ8ZSVLt8o/wOfWY1ac6ZjvOwr8=' 'sha256-BeZ6EL8ajstckTUyE8k0bjY3v2P92XLxVp25GnrBG30=' 'sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=';";
+        add_header Content-Security-Policy "default-src 'self'; frame-ancestors 'self'; img-src 'self' data: https://openmaps.gov.bc.ca/ https://maps.gov.bc.ca/ https://server.arcgisonline.com/; frame-src 'self' https://*.oidc.gov.bc.ca; connect-src 'self' https://*.oidc.gov.bc.ca/ https://openmaps.gov.bc.ca/ https://maps.gov.bc.ca/ https://server.arcgisonline.com/; form-action 'self'; style-src 'self' 'sha256-TJWtAQZhIjMP4+e4lQ8ZSVLt8o/wOfWY1ac6ZjvOwr8=' 'sha256-BeZ6EL8ajstckTUyE8k0bjY3v2P92XLxVp25GnrBG30=' 'sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=';";
```

Comment:
Updated the CSP for OpenShift deployments

Reply:
If I'm remembering correctly this may require a manual build of the bc that builds the nginx conf. Worth following up in dev to make sure either way.

Reply:
I see. I think you are right. Will trigger a manual rebuild of nginx then